cloudbuild.yaml

```diff
@@ -14,7 +14,6 @@ steps:
       - kueueviz-image-push
     env:
       - IMAGE_REGISTRY=us-central1-docker.pkg.dev/k8s-staging-images/kueue
-      - GIT_TAG=$_GIT_TAG
```

Comment:
After removing `GIT_TAG` here, the internal Makefile value from `git describe --tags --dirty --always` will be used instead, so it should still work fine. However, the issue is that we'll lose the `v20250526` date prefix from the image tag.

$_GIT_TAG includes the date prefix (e.g., `v20250526-v0.12.0`), while the Git version alone would be just `v0.12.0`. To stay consistent with the current approach, perhaps instead of using `GIT_TAG`, we could introduce a separate variable like `IMAGE_TAG` or `GIT_TAG_EXTENDED` that includes the date prefix.

But if it's acceptable to use only the v0.12.0 image tag, then I think it should work fine.

Reply:
cc @mimowo @tenzen-y

Reply:
> For (1), what if release-branch git version?

In GIT_VERSION from cloudbuild we have v20250526-v0.12.0. Locally GIT_BRANCH is v0.12.0 which is weird. But we have EXTRA_TAG variable in cloudbuild to build main or release-0.12 tags.

Reply:
One potential complication is that `EXTRA_TAG=$_PULL_BASE_REF` which may also be pointing to tags. I think this is another issue, but I think it will be better to use EXTRA_TAG=git branch --show-current.

Reply:
I'm ok with dropping the date prefix from the image tag.
Thank you.

/lgtm
/approve

Reply:
> EXTRA_TAG=git branch --show-current
no values taken from cloudbuild.

I think the extra tag from cloudbuild is fine. We don't need this variable during the release process.

Reply:
> So, I argue we should delete it. Just use GIT_TAG inside push-charts.sh

This is not correct. We need to push both tags.